### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by AnisZamri** [rappasoft/laravel-livewire-tables#1227] [Q&A] [answered]
Below are the code i have successfully translated using basic php, but i dont have any idea on how to translate the display_name value in livewire table. Anyone can help?
![livewire table](https://github.com/rappasoft/laravel-livewire-tables/assets/93184571/56ff83ba-963c-4732-93b6-4c522974c9f4)
![basic php](https://github.com/rappasoft/laravel-livewire-tables/assets/93184571/2b6487d6-ad92-47a2-9c32-914da5f84fa9)

**Comment by lrljoe** [comment, marked as answer]
If you look at the format() option in the documentation, then this covers off how to change the result that will be displayed for a given Column.

 it's in the available methods section of the documentation for Columns

**Reply by AnisZamri** [reply]
thanks! its working
